PartsApp/SaleForm.cs

```diff
@@ -944,6 +944,22 @@ private void markupComboBox_SelectedIndexChanged(object sender, EventArgs e)
         //////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
         #endregion        
 
+        private void OnPaymentTypeCheckBoxClick(object sender, EventArgs e)
+        {
+            var changedCheckBox = sender as System.Windows.Forms.CheckBox;
+            if (changedCheckBox.Checked)
+            {
+                if (changedCheckBox == PaidCashCheckBox)
+                {
+                    PaidNonCashCheckBox.Checked = false;
+                }
+                else if (changedCheckBox == PaidNonCashCheckBox)
+                {
+                    PaidCashCheckBox.Checked = false;
+                }
+            }
+        }
+
```

Comment:
Why would you possibly need two check boxes for such a simple scenario?

Reply:
removed redundant 'PaidCash' checkbox